### PR TITLE
Update Sky Atlas for Oct 14 poll

### PR DESCRIPTION
**Pull request by MakerJanSky**
DO NOT MERGE UNLESS POLL PASSES

TODO
- [ ] Review Formatting
- [ ] Obtain AA approval

### CODE REVIEW

File: Sky Atlas/Sky Atlas.html [hunk 1]
```diff
@@ -6930,225 +6930,18 @@ <h1>Sections & Primary Docs</h1>
...
+              - A1 - Surplus Buffer And Smart Burn Engine - Surplus Buffer Splitter Parameters</dfn>
           </td>
-          <td>Surplus Buffer And Smart Burn Engine - Surplus Buffer Upper Limit</td>
+          <td>Surplus Buffer And Smart Burn Engine - Surplus Buffer Splitter Parameters</td>
           <td>Section</td>
-          <td>This Section defines the Surplus Buffer Upper Limit and key parameters of the Smart Burn Engine.
+          <td>The current Surplus Buffer Splitter parameters are:• splitter.hump: 60m (Threshold of Surplus buffer for Splitter to activate)•
+              Estimated Net protocol Income: 100,765,523• 50% of estimated net protocol income is sent to Splitter: 50,382,762 (for SBE and 
+              Seal Module, remaining 50% is accumulated in Surplus Buffer). This value must be regularly updated through an executive vote to 
+              continue to target 50% of the estimated net protocol income.• splitter.bump: 25,000 DAI• splitter.hop: 15,649 seconds• 70% of
```
Comment (0xldr) [inline Sky Atlas/Sky Atlas.html:6940]: bump and hump are vow parameters.
```suggestion
          <td>The current Surplus Buffer Splitter parameters are:• vow.hump: 60m (Threshold of Surplus buffer for Splitter to activate)•
              Estimated Net protocol Income: 100,765,523• 50% of estimated net protocol income is sent to Splitter: 50,382,762 (for SBE and 
              Seal Module, remaining 50% is accumulated in Surplus Buffer). This value must be regularly updated through an executive vote to 
              continue to target 50% of the estimated net protocol income.• vow.bump: 25,000 DAI• splitter.hop: 15,649 seconds• 70% of 
```

File: Sky Atlas/Sky Atlas.html [hunk 2]
```diff
@@ -10723,12 +10564,12 @@ <h1>Annotations</h1>
         </tr>
         <tr>
           <td>
-            <dfn>Total
-              Votes Delegated - Element Annotation</dfn>
+            <dfn>Total Voting Power Delegated - Element Annotation</dfn>
           </td>
-          <td>Total Votes Delegated - Element Annotation</td>
+          <td>Total Voting Power Delegated  - Element Annotation</td>
```
Comment (0xldr) [inline Sky Atlas/Sky Atlas.html:10569]:
```suggestion
          <td>Total Voting Power Delegated - Element Annotation</td>
```

File: Sky Atlas/Sky Atlas.html [hunk 3]
```diff
@@ -1991,9 +2006,8 @@ <h1>Sections & Primary Docs</h1>
           <td>Aligned Delegates - Budget And Participation Requirements - Prime Delegates</td>
           <td>Section</td>
           <td>
-            Aligned Delegates (ADs) who receive a budget are designated as Prime Delegates (PDs). The PD slots
-            are allocated to the ADs with the highest number of delegated votes. The number of PD slots is
-            specified in <dfn>A.1.5
+            Aligned Delegates with the greatest delegated Voting Power are designated as Prime Delegates and receive a budget. 
+            The number of Prime Delegate slots is specified in  <dfn>A.1.5
```
Comment (blimpa) [inline Sky Atlas/Sky Atlas.html:2010]:
```suggestion
            The number of Prime Delegate slots is specified in <dfn>A.1.5
```

File: Sky Atlas/Sky Atlas.html [hunk 4]
```diff
@@ -2057,10 +2070,23 @@ <h1>Sections & Primary Docs</h1>
           </td>
           <td>Aligned Delegates - Budget And Participation Requirements - AD Buffer And Loss Of Budget Stream</td>
           <td>Section</td>
-          <td>If an AD loses a part of its budget stream due to a transition from Prime Delegate (PD) to Aligned
-            Delegate (AD), their AD Buffer will stay paying out its excess funds until the remaining balance
-            equals one month's budget allocation. If an AD loses its entire budget stream, the full contents of
-            the AD Buffer can be spent.</td>
+          <td>If an AD loses its budget due to losing its Prime Delegate rank, the full contents of the AD Buffer 
+            can be paid out in the next AD compensation cycle**,** except as specified in A.1.5 - Aligned Delegates - 
+            Budget And Participation Requirements - Payout Limitations For ADs Triggering Atlas Edit Weekly Cycle Proposals.</td>
```
Comment (blimpa) [inline Sky Atlas/Sky Atlas.html:2075]:
```suggestion
            Budget And Participation Requirements - Payout Limitations For ADs Triggering Atlas Edit Weekly Cycle Proposals</dfn>.</td>
```

File: Sky Atlas/Sky Atlas.html [hunk 5]
```diff
@@ -2057,10 +2070,23 @@ <h1>Sections & Primary Docs</h1>
...
+            <dfn>A.1.5
+              - Aligned Delegates - Budget And Participation Requirements - AD Buffer And Loss Of Budget
+              Stream - Payout Limitations For ADs Triggering Atlas Edit Weekly Cycle Proposals</dfn>
+          </td>
+          <td>Aligned Delegates - Budget And Participation Requirements - AD Buffer And Loss Of Budget Stream - Payout Limitations 
+            For ADs Triggering Atlas Edit Weekly Cycle Proposals</td>
+          <td>Section</td>
+          <td>If an AD has triggered an Atlas Edit Weekly Cycle Proposal pursuant to A.1.11 - Weekly Governance Cycle - Atlas Edit 
+            Weekly Cycle - Triggering Requirement, no payouts may be made that would reduce their AD Buffer below one (1) month’s 
+            worth of the budget specified in A.1.5 - Aligned Delegates - Budget And Participation Requirements - Budget Amount For
```
Comment (blimpa) [inline Sky Atlas/Sky Atlas.html:2088]:
```suggestion
            worth of the budget specified in <dfn>A.1.5 - Aligned Delegates - Budget And Participation Requirements - Budget Amount For 
```

File: Sky Atlas/Sky Atlas.html [hunk 6]
```diff
@@ -2057,10 +2070,23 @@ <h1>Sections & Primary Docs</h1>
...
+              - Aligned Delegates - Budget And Participation Requirements - AD Buffer And Loss Of Budget
+              Stream - Payout Limitations For ADs Triggering Atlas Edit Weekly Cycle Proposals</dfn>
+          </td>
+          <td>Aligned Delegates - Budget And Participation Requirements - AD Buffer And Loss Of Budget Stream - Payout Limitations 
+            For ADs Triggering Atlas Edit Weekly Cycle Proposals</td>
+          <td>Section</td>
+          <td>If an AD has triggered an Atlas Edit Weekly Cycle Proposal pursuant to A.1.11 - Weekly Governance Cycle - Atlas Edit 
+            Weekly Cycle - Triggering Requirement, no payouts may be made that would reduce their AD Buffer below one (1) month’s 
+            worth of the budget specified in A.1.5 - Aligned Delegates - Budget And Participation Requirements - Budget Amount For 
+            Prime Delegate Slots until the Proposal has been voted on or rejected by the Governance Facilitators for misalignment.</td>
```
Comment (blimpa) [inline Sky Atlas/Sky Atlas.html:2089]:
```suggestion
            Prime Delegate Slots</dfn> until the Proposal has been voted on or rejected by the Governance Facilitators for misalignment.</td>
```

File: Sky Atlas/Sky Atlas.html [hunk 7]
```diff
@@ -2057,10 +2070,23 @@ <h1>Sections & Primary Docs</h1>
...
+        <tr>
+          <td>
+            <dfn>A.1.5
+              - Aligned Delegates - Budget And Participation Requirements - AD Buffer And Loss Of Budget
+              Stream - Payout Limitations For ADs Triggering Atlas Edit Weekly Cycle Proposals</dfn>
+          </td>
+          <td>Aligned Delegates - Budget And Participation Requirements - AD Buffer And Loss Of Budget Stream - Payout Limitations 
+            For ADs Triggering Atlas Edit Weekly Cycle Proposals</td>
+          <td>Section</td>
+          <td>If an AD has triggered an Atlas Edit Weekly Cycle Proposal pursuant to A.1.11 - Weekly Governance Cycle - Atlas Edit
```
Comment (blimpa) [inline Sky Atlas/Sky Atlas.html:2086]:
```suggestion
          <td>If an AD has triggered an Atlas Edit Weekly Cycle Proposal pursuant to <dfn>A.1.11 - Weekly Governance Cycle - Atlas Edit 
```

File: Sky Atlas/Sky Atlas.html [hunk 8]
```diff
@@ -2057,10 +2070,23 @@ <h1>Sections & Primary Docs</h1>
...
+          <td>
+            <dfn>A.1.5
+              - Aligned Delegates - Budget And Participation Requirements - AD Buffer And Loss Of Budget
+              Stream - Payout Limitations For ADs Triggering Atlas Edit Weekly Cycle Proposals</dfn>
+          </td>
+          <td>Aligned Delegates - Budget And Participation Requirements - AD Buffer And Loss Of Budget Stream - Payout Limitations 
+            For ADs Triggering Atlas Edit Weekly Cycle Proposals</td>
+          <td>Section</td>
+          <td>If an AD has triggered an Atlas Edit Weekly Cycle Proposal pursuant to A.1.11 - Weekly Governance Cycle - Atlas Edit 
+            Weekly Cycle - Triggering Requirement, no payouts may be made that would reduce their AD Buffer below one (1) month’s
```
Comment (blimpa) [inline Sky Atlas/Sky Atlas.html:2087]:
```suggestion
            Weekly Cycle - Triggering Requirement</dfn>, no payouts may be made that would reduce their AD Buffer below one (1) month’s 
```

**Review by blimpa — CHANGES_REQUESTED** [review]
dfn tags

**Review by 0xldr — APPROVED** [review]

**Pull request merged**